.kitchen.yml

```diff
@@ -12,19 +12,41 @@ provisioner:
 
 platforms:
   - name: ubuntu-12.04
-  - name: centos-6.4
+  - name: centos-7.1
 
 suites:
-  - name: default
+  - name: rhel_init
```

Comment:
I really didn't want to, but I thought including the `apt` recipe under CentOS would break. Tested this and it did create the `/etc/apt` directories and other cruft, but it didn't try to execute so it didn't fail. I'll flatten this back out to `default`.